scripts/s3mi

```diff
@@ -353,10 +354,15 @@ def get_credentials():
     [Acknowledgement:  Thanks Andrey Kislyuk, David Rissato Cruz]
     """
     if not have_boto:
-        # This effectively disables credentials caching/update.
-        return {"vars": {}, "expiration_time": time.time() + 365*24*3600}
+        tsprint("Refreshing credentials is impossible without boto.  Disabling.")
```

Comment:
May I suggest writing `not supported` instead of `impossible`?

Reply:
Yes that is better wording, thanks.